provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerTransformer.java

```diff
@@ -49,25 +51,34 @@ Address address(ProviderResponse.Value value) {
 
   @Override
   public Practitioner apply(PractitionerWrapper ppmsData) {
-    ProviderResponse.Value response = ppmsData.providerResponse().value().get(0);
-    ProviderContactsResponse.Value contacts;
-    if (ppmsData.providerContactsResponse().value().size() > 0) {
-      contacts = ppmsData.providerContactsResponse().value().get(0);
-    } else {
-      contacts = null;
-    }
+    ProviderResponse.Value providerResponse = ppmsData.providerResponse().value().get(0);
+    ProviderContactsResponse.Value providerContacts =
+        ppmsData.providerContactsResponse().value() == null
+                || ppmsData.providerContactsResponse().value().isEmpty()
+            ? null
+            : ppmsData.providerContactsResponse().value().get(0);
+    ProviderServicesResponse.Value providerServices =
+        ppmsData.providerServicesResponse().value() == null
+                || ppmsData.providerServicesResponse().value().isEmpty()
+            ? null
+            : ppmsData.providerServicesResponse().value().get(0);
     List<Practitioner.PractitionerIdentifier> identifiers = new ArrayList<>();
-    identifiers.add(identifier(response));
+    identifiers.add(identifier(providerResponse));
```

Comment:
As a small simplification, you can eliminate `identifiers` altogether by inlining this list down below as `Arrays.asList(identifier(providerResponse))`